### PR TITLE
New TU Delft faculty members (multiple csrankings files)

### PR DESCRIPTION
**Pull request by gsmaragd**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [ X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [X ] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [X ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X ] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [X ] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-h.csv [hunk 1]
```diff
@@ -27,6 +27,7 @@ Hadas Shachnai,Technion,http://www.cs.technion.ac.il/~hadas,NOSCHOLARPAGE
 Hadi Amiri,University of Massachusetts Lowell,https://scholar.harvard.edu/hadi,fyUpZ5EAAAAJ
 Hadi Hemmati,University of Manitoba,http://sealab.cs.umanitoba.ca/?page_id=149,TznXpSIAAAAJ
 Hadi Hosseini,Pennsylvania State University,https://faculty.ist.psu.edu/hadi,bnvEtsMAAAAJ
+Hadi Jamali-Rad,TU Delft,https://www.tudelft.nl/en/ewi/over-de-faculteit/afdelingen/intelligent-systems/pattern-recognition-bioinformatics/computer-vision-lab/people/hadi-jamali-rad,l4hw34oAAAAJ
```
Comment (emeryberger) [inline csrankings-h.csv]: Web page says he only has a 20% (part-time) appointment. Please remove, per the instructions.

**Review by emeryberger — CHANGES_REQUESTED** [review]
See comment.

File: csrankings-h.csv (hunk 1 above)
Comment (gsmaragd) [reply csrankings-h.csv]: I did not notice that, thx! Faculty member was removed.

**Comment by gsmaragd** [comment]
I added some new faculty members.

Thomas Durieux has joined TU Delft CS (software engineering group recently from KTH)
https://se.ewi.tudelft.nl/people.html

File: csrankings-t.csv [hunk 2]
```diff
@@ -357,6 +357,7 @@ Thomas Courtade,Univ. of California - Berkeley,https://people.eecs.berkeley.edu/
 Thomas D. LaToza,George Mason University,https://cs.gmu.edu/~tlatoza,200KNLAAAAAJ
 Thomas D. Nielsen,Aalborg University,http://people.cs.aau.dk/~tdn,6fWF0CgAAAAJ
 Thomas Dreier,Karlsruhe Institute of Technology,http://www.zar.kit.edu/591.php?ID=3,NOSCHOLARPAGE
+Thomas Durieux,TU Delft,https://durieux.me/,CoUKVbUAAAAJ
```
Comment (emeryberger) [inline csrankings-t.csv:360]: No indication of Delft affiliation.

File: csrankings-l.csv [hunk 3]
```diff
@@ -794,6 +794,8 @@ Luís Caldas de Oliveira,Universidade de Lisboa,https://www.l2f.inesc-id.pt/wiki
 Luís Carlos Erpen De Bona,UFPR,https://web.inf.ufpr.br/bona,YJL1PhUAAAAJ
 Luís Carriço,Universidade de Lisboa,https://ciencias.ulisboa.pt/perfil/lmcarrico,MLg1ZJMAAAAJ
 Luis Ceze,University of Washington,https://www.cs.washington.edu/people/faculty/luisceze,KzESVKwAAAAJ
+Luis Cruz,TU Delft,https://luiscruz.github.io/,O13oaH0AAAAJ
```
Comment (emeryberger) [inline csrankings-l.csv]: Looks like wrong DBLP: see https://dblp.org/pid/96/2486-2.html.

File: csrankings-b.csv [hunk 4]
```diff
@@ -225,6 +225,7 @@ Ben Stock,CISPA Helmholtz Center,https://people.cispa.io/ben.stock,_hDa-5wAAAAJ
 Ben Swift,Australian National University,https://cs.anu.edu.au/people/ben-swift,OQdYgLEAAAAJ
 Ben W. Reichardt,University of Southern California,http://www-bcf.usc.edu/~breichar,BEGRf_EAAAAJ
 Ben Y. Zhao,University of Chicago,http://people.cs.uchicago.edu/~ravenben,cYReSuEAAAAJ
+Benedikt Ahrens,TU Delft,https://benediktahrens.gitlab.io/,hdr4g6UAAAAJ
```
Comment (emeryberger) [inline csrankings-b.csv:228]: Clarify appointment level at Delft - web page indicates dual appointment.

**Review by emeryberger — CHANGES_REQUESTED** [review]
See comments.

File: csrankings-t.csv (hunk 2 above)
Comment (gsmaragd) [reply csrankings-t.csv:360]: Thomas Durieux has joined TU Delft CS (software engineering group recently from KTH)
https://se.ewi.tudelft.nl/people.html

File: csrankings-l.csv (hunk 3 above)
Comment (gsmaragd) [reply csrankings-l.csv]: Should I add alias "Luis Cruz 0002" and "Luís Cruz 0002"?

File: csrankings-b.csv (hunk 4 above)
Comment (gsmaragd) [reply csrankings-b.csv:228]: Benedikt is Assistant Prof.  (full time) at TU Delft, see: https://benediktahrens.gitlab.io/cv/
The other is just a secondary affiliation.

**Comment by gsmaragd** [comment]
I hope, I addressed all your comments.

File: csrankings-l.csv (hunk 3 above)
Comment (emeryberger) [reply csrankings-l.csv]: You should only have entries corresponding to the above aliases.

**Comment by gsmaragd** [comment]
"Luis Cruz 0002 and Luís Cruz 0002 for DBLP entrance: https://dblp.org/pid/96/2486-2.html"

File: csrankings-l.csv (hunk 3 above)
Comment (emeryberger) [reply csrankings-l.csv]: Please remove the "old" entries for Luis Cruz (the ones without the disambiguation suffixes).

**Comment by gsmaragd** [comment]
keeping only Luis Cruz 0002 and Luís Cruz 0002 as appears in DBLP entrance: https://dblp.org/pid/96/2486-2.html

**Pull request merged**